modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/bulk_copy_entries.jsp

```diff
@@ -0,0 +1,35 @@
+<%--
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+--%>
+
+<%@ include file="/document_library/init.jsp" %>
+
+<%
+DLCopyEntryDisplayContext dlCopyEntryDisplayContext = new DLCopyEntryDisplayContext(request, liferayPortletResponse, themeDisplay);
+
+dlCopyEntryDisplayContext.setViewAttributes();
+%>
+
+<div class="c-mt-3 sheet sheet-lg">
+	<react:component
+		module="document_library/js/DLFolderSelector"
+		props='<%=
+			HashMapBuilder.<String, Object>put(
+				"copyActionURL", dlCopyEntryDisplayContext.getCopyEntriesActionURL()
+			).put(
+				"itemType", "fileEntry"
+			).put(
+				"redirect", dlCopyEntryDisplayContext.getRedirect()
+			).put(
+				"selectedEntries", dlCopyEntryDisplayContext.getSelectedEntries()
+			).put(
+				"selectionModalURL", dlCopyEntryDisplayContext.getSelectFolderURL()
+			).put(
+				"sourceFileName", dlCopyEntryDisplayContext.getSourceFolderName()
```

Comment:
Why this mismatch between the parameter name and the input method name? Is it an error?

Reply:
No idea. I've seen other weird things in the frontend code; I think it was initially written to copy only files, and the naming is a bit wonky. We'll need to revisit this once Ambrin is back (and before sending it to Brian). For the moment I'm leaving it as it is.

Reply:
Hey guys, yeah this component is getting bigger and strange.... 
First we have only the possibility to copy files (`copy_file_entry.jsp`), then we added the copy for folders and created a new jsp (`copy_folder.jsp`) instead of integrating it, and after that this `DLFolderSelector.js` was created "appending" the copy for file entries and folders (so you have different parameters `sourceFileName`, `sourceFolderName`, `sourceFileEntryId` and `sourceFolderId`). 
Now we are creating a new `bulk_copy_entries.jsp` which uses the same component. I don't this this is the good approach, can't we mix them and just have one .jsp and one .js component? Maybe we can work on that in a different task, but then at least, here you should use `sourceFolderName` parameter instead of `sourceFileName` for consistency, don't you think?

Reply:
Sure. I'm working on that right now.